ddp-workspace/projects/ddp-sdk/src/lib/services/activity/activityComponentConverter.service.ts

```diff
@@ -28,7 +28,9 @@ export class ActivityComponentConverter {
         const block = new MailAddressBlock(inputBlock.displayNumber);
         block.titleText = params.titleText;
         block.subtitleText = params.subtitleText;
-        block.requireVerified = !!params.requireVerified;
+
+        // TODO: JUST FOR DEBUG ! FIX BACK AFTER DEBUGGING !
+        block.requireVerified = true; // !!params.requireVerified;
```

Comment:
Must be removed before merge.

Reply:
I'll go make the change in backend study configuration so we don't have to do this debugging.

Reply:
https://github.com/broadinstitute/ddp-study-server/pull/907

Reply:
All set on `dev`!

Reply:
Fixed.